README.md

```diff
@@ -1 +1,211 @@
-# Budget-App
\ No newline at end of file
+# Budget Tracker App
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  <!-- - [🚀 Live Demo](#live-demo) -->
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+
+# 📖 Budget Tracker App <a name="about-project"></a>
+
+> The Budget Tracker app is a web application that allows users to track their personal finances. Users can create an account to save their expense data over time.The project focuses on backend Ruby on Rails architecture, database structure, RESTful routing, and core money tracking functionality. The minimal frontend uses Embedded Ruby for dynamic updates.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Here is a list of technogies used in this project.
+
+<details>
+  <summary>Language</summary>
+  <ul>
+    <li><a href="https://rubylang.org/">Ruby</a></li>
+    <li><a href="https://developer.mozilla.org/">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+
+### Key Features <a name="key-features"></a>
+
+- User authentication
+- Income/expense tracking
+- Categorize expenses
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming soon...]()
```

Comment:
- [ ] [REQUIRED] Kindly ensure that you deploy the application using Render, and Netlify so that end users can easily access your app. This is also a hard requirement.